modules/weko-schema-ui/weko_schema_ui/schema.py

```diff
@@ -820,6 +820,33 @@ def merge_json_xml(json, dct):
                     k = 'jpcoar:file'
                 elif k == 'custom:system_identifier':
                     k = 'jpcoar:identifier'
+
+                # Remove items that are not set as controlled vocabulary
+                elif k == 'jpcoar:creator' or k == 'jpcoar:contributor' \
+                        or k == 'jpcoar:rightsHolder':
+                    if '@attributes' in v['jpcoar:nameIdentifier']:
+                        lst_nameIdentifierScheme = v['jpcoar:nameIdentifier'][
+                            '@attributes']['nameIdentifierScheme'][0]
```

Comment:
Plz add var config for 0 to prevent checklist: Do you use a magic number?

Thanks,